graphql-server/src/schemas/neo4j.ts

```diff
@@ -42,9 +47,39 @@ export async function buildNeo4jDbSchema() {
   });
 
   const typeDefs = await toGraphQLTypeDefs(sessionFactory, false);
-  const ogm = new OGM({ typeDefs: typeDefs, driver });
+  const extendedTypeDefs = gql`
+    ${typeDefs}
+
+    extend type Request {
+      totalSampleCount: Int
+    }
+
+    extend type Patient {
+      cmoPatientId: String
+      dmpPatientId: String
+      totalSampleCount: Int
+      cmoSampleIds: [String]
+      consentPartA: String
+      consentPartC: String
+    }
+
+    extend type Cohort {
+      totalSampleCount: Int
+      smileSampleIds: [String]
+      billed: String
+      initialCohortDeliveryDate: String
+      endUsers: String
+      pmUsers: String
+      projectTitle: String
+      projectSubtitle: String
+      status: String
+      type: String
+    }
+  `;
```

Comment:
Added extra fields to the schema. This let us bring these nested fields to the root level of a node.